### ISSUE TITLE
SuccessStory does not function properly with RPSC3

### ISSUE THREAD

**Issue by Noclue12** [Lacro59/playnite-successstory-plugin#522]
SuccessStory does not function properly with RPSC3 for me.
I'm pretty sure I've got everything set up right.
It appears that the plugin does not see that I have unlocked achievements in Games on RPSC3, despite feeding SuccessStory with the proper link.

anyone got any idea on why this could be the case? Thanks.

![Screenshot 2024-09-03 152846](https://github.com/user-attachments/assets/fb3b7d6f-1668-4190-9398-08f1c87195b1)
![Screenshot 2024-09-03 155242](https://github.com/user-attachments/assets/b1e32977-74b9-4f18-8ac2-a35f565c5fdb)

**Comment by Noclue12** [comment]
yeah after further testing with other games it seems to come down to:  No TrueAchievements url in the Json file of the game. how can this be fixed?

**Comment by Lacro59** [comment]
The issue comes from reading the achievements.
Can you post a TROPUSR.DAT file?

**Comment by Noclue12** [comment]
Here is the TROPUSR.DAT from E.X Troopers (エクストルーパーズ) 

[TROPUSR.zip](https://github.com/user-attachments/files/19398233/TROPUSR.zip)

**Comment by Noclue12** [comment]
And Here is the one from Time and Eternity

[TROPUSR Time.zip](https://github.com/user-attachments/files/19398244/TROPUSR.Time.zip)

**Comment by Lacro59** [comment]
Can you try this version?
[playnite-successstory-plugin_3_4_1.zip](https://github.com/user-attachments/files/19760313/playnite-successstory-plugin_3_4_1.zip)

**Comment by Lacro59** [comment]
@Noclue12 , sorry.
I've uploaded a debug version for GameJolt...
This one will be better.
[playnite-successstory-plugin_3_4_1.zip](https://github.com/user-attachments/files/19813897/playnite-successstory-plugin_3_4_1.zip)

**Comment by Noclue12** [comment]
Lacro59, thank you so much! im thankful for your work. i still cant get RPSC3 achievements to work. here are two more log files: 
[extensions.zip](https://github.com/user-attachments/files/19847537/extensions.zip)
[SuccessStory_2025-04-22.zip](https://github.com/user-attachments/files/19847589/SuccessStory_2025-04-22.zip)

As a side note, seeing how ZZZ trophies were implemented was quite surprising. That's Awesome! Is there any documentation on how to properly configure them yet? I clicked add. It found 400+ achievements. But I suppose the import feature is of importance to properly track them yes? Where do I find the JSON that it is looking for? 

Thank you so much.
Keep up the great work

**Comment by Lacro59** [comment]
@Noclue12 You can find more information [here](https://github.com/Lacro59/playnite-successstory-plugin/wiki/Configure-gatcha-games#zenless-zone-zero).
